src/views/pages/ManagePeople/index.vue

```diff
@@ -52,81 +53,73 @@ export default Vue.extend({
       this.searched = searchByName(this.people, this.search);
     },
   },
-  created: function created() {
-    this.searched = this.people;
+  created: async function created() {
+    const self = this;
+
+    // get people types
```

Comment:
What is axios just an abstraction over AJAX? This exists natively in our environment w/ `fetch`